django_socio_grpc/tests/test_async_enum_service.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase, override_settings
+from fakeapp.grpc import fakeapp_pb2
+from fakeapp.grpc.fakeapp_pb2_grpc import (
+    EnumControllerStub,
+    add_EnumControllerServicer_to_server,
+)
+from fakeapp.services.enum_service import EnumService
+
+from .grpc_test_utils.fake_grpc import FakeFullAIOGRPC
+
+
+@override_settings(GRPC_FRAMEWORK={"GRPC_ASYNC": True})
+class TestAsyncEnumService(TestCase):
+    def setUp(self):
+        self.fake_grpc = FakeFullAIOGRPC(
+            add_EnumControllerServicer_to_server, EnumService.as_servicer()
+        )
+
+    def tearDown(self):
+        self.fake_grpc.close()
+
+    async def test_async_enum_service_with_annotated_model(self):
+        grpc_stub = self.fake_grpc.get_fake_stub(EnumControllerStub)
+        request = fakeapp_pb2.EnumServiceRequest(char_choices=1, int_choices=2)
+        response = await grpc_stub.BasicEnumRequestWithAnnotatedModel(request=request)
+        self.assertEqual(response.char_choices, 1)
+        self.assertEqual(response.int_choices, 2)
+
+    async def test_async_enum_service_with_annotated_serializer(self):
```

Comment:
It needs a test that the serializer conversion works, if I call the a model creation with MyGRPCActionEnum.VALUE_1 I should have `VALUE_1` in my DB as a result. And if I retrieve it should also match MyGRPCActionEnum.VALUE_1